### DISCUSSION TITLE
Welcome to #RSEAA2023 Posters!!!

### DISCUSSION THREAD

**Discussion by rowlandm** [RSEAA/RSEAA.github.io#32] [Announcements]
## 👋 Welcome!
  We’re using GitHub Discussions as a place for #RSEAA2023 posters as well as to guide the discussion topics of our first, second and third sessions of the unconference. 

## Posters
We hope that you would like to contribute by creating a poster by:
1.  uploading an accessible powerpoint document / image / pdf to figshare and adding the keyword #RSEAA2023 to the figshare item
2. creating a discussion under the category "Posters for #RSEAA2023" on this board and putting the link to the figshare item

This is [an example of a poster from a figshare item that was made earlier.](https://github.com/RSEAA/RSEAA.github.io/discussions/37)

The idea of the poster would be to possibly:
  * introduce yourself 
  * share where you work 
  * ask a question of the community or a talking point - "What tool are people using for X?"
  * suggest a solution  - "We have had this problem X and we solved it with Y"

... based on whatever you are comfortable with.

### Look through the current Posters

You can see all the discussion topics by:
1. [latest activity ](https://github.com/RSEAA/RSEAA.github.io/discussions/categories/posters-for-rseaa2023?discussions_q=is%3Aopen+category%3A%22Posters+for+%23RSEAA2023%22) or
2. [most popular](https://github.com/RSEAA/RSEAA.github.io/discussions/categories/posters-for-rseaa2023?discussions_q=is%3Aopen+category%3A%22Posters+for+%23RSEAA2023%22+sort%3Atop+created%3A%3E%3D2022-08-04) or
3. [view directly in figshare](https://figshare.com/search?q=:keyword:%20%22%23RSEAA2023%22)

## Discussion Topics

We also [have Discussion Topics that will guide our discussions at the unconference](https://github.com/RSEAA/RSEAA.github.io/discussions/43).

**Comment by chesadi** [comment]
This is the Poster.
Anusha
[V3_Poster_W.A.Indika.pdf](https://github.com/RSEAA/RSEAA.github.io/files/12604808/V3_Poster_W.A.Indika.pdf)

**Reply by rowlandm** [reply]
Could you [please go to the github discussion poster area](https://github.com/RSEAA/RSEAA.github.io/discussions/categories/posters-for-rseaa2023?discussions_q=is%3Aopen+category%3A%22Posters+for+%23RSEAA2023%22), then create "New Discussion". Then you can add in this poster.
